### ISSUE TITLE
ETH has a hole in it

### ISSUE THREAD

**Issue by jbpanganiban** [wmgeolab/gbRelease#31]
The holes are attributed to its geography. Features such as Lake Tana multiple lower-level administrative divisions. In the future, we plan to improve our boundaries to dictate which administrative division each section of the features falls under.

Ethiopia with OpenStreetMap base 
![image](https://user-images.githubusercontent.com/61036228/83693067-df797300-a5ba-11ea-87ab-ea567ea9c724.png)

**Comment by dickoa** [comment]
Hi @DanRunfola,

Is it possible to check the latest COD-AB of Ethiopia with data source from government and shared in HDX as a COD. I think it solves the hole issue (there's no hole) and it's updated with more features at adm2 and adm3.

https://data.humdata.org/dataset/ethiopia-cod-ab

You have more features than what is available under the version 3.0.0 of geoBoudaries.

```r
> layers
Driver: ESRI Shapefile 
Available layers:
                                  layer_name geometry_type
1        eth_admbnda_adm0_csa_bofed_20190827       Polygon
2        eth_admbnda_adm1_csa_bofed_20190827       Polygon
3        eth_admbnda_adm2_csa_bofed_20190827       Polygon
4        eth_admbnda_adm3_csa_bofed_20190827       Polygon
5 eth_admbndl_admALL_csa_bofed_itos_20190827   Line String
6 eth_admbndp_admALL_csa_bofed_itos_20190827         Point
7                eth_admbndt_adminUnitLookup            NA
  features fields
1        1     10
2       11     12
3       91     14
4      981     16
5     2785      5
6      981     13
7        4      6
```
Thanks

**Comment by DanRunfola** [comment]
Hi Ahmadou,

I actually just got off the phone with HDX, and we're going to explore
integrating HDX boundaries into geoBoundaries programmatically in the very
near future.  I'm going to keep this ticket open pending that, but the goal
will be to provide a single unified global product using the more formal
HDX boundaries + our operational boundaries for other countries.

More information on that is coming soon - probably realistically aiming for
~4.0.
